.github/workflows/create-release.yml

```diff
@@ -170,92 +170,35 @@ jobs:
           echo 'release-kyma-metrics-collector-build failed.'
           exit 1
 
-  bump-kmc-chart:
-    name: Bump KMC chart
-    needs: [ bump-sec-scanners-config, wait-for-prow-jobs ]
+  check-prerequisites:
```

Comment:
why this file is changed?

Reply:
Now we are not packaging the helm chart and uploading it to the release.